litestar/stores/redis.py

```diff
@@ -21,7 +21,7 @@
 class RedisStore(NamespacedStore):
     """Redis based, thread and process safe asynchronous key/value store."""
 
-    __slots__ = ("_redis",)
+    __slots__ = ("_redis", "handle_client_shutdown", "_get_and_renew_script", "_delete_all_script")
```

Comment:
probably a dumb question but why do we expose private methods here

Reply:
note that `__slots__` isn't about exposing things like `__all__` is. `__slots__` merely tells Python to only allow certain attributes on the class in order to optimize memory. Private attributes are no exception.

Reply:
ahh yep.. i thought this was __all__ i think i got lost in the buzz.